src/ripple/protocol/SystemParameters.h

```diff
@@ -46,7 +46,7 @@ constexpr XRPAmount INITIAL_XRP{100'000'000'000 * DROPS_PER_XRP};
 inline bool
 isLegalAmount(XRPAmount const& amount)
 {
-    return amount <= INITIAL_XRP;
+    return amount >= -INITIAL_XRP && amount <= INITIAL_XRP;
```

Comment:
Why? This may have implications in other parts of the code.

Reply:
Yeah, on second thought, this may be the wrong way / place to implement this. The reason is that somewhere in the development process, I ran into something where a negative `XRPAmount` was possible, and could have caused trouble. In other words, I can't imagine negative one-gazillion XRP would be any better than positive one-gazillion XRP. But I'll reexamine this.

Reply:
Fixed.